CONTRIBUTING.md

```diff
@@ -36,6 +36,14 @@ git clone https://github.com/pytorch/pytorch
 cd pytorch
 ```
 
+2.1. If you already have PyTorch from source, update it:
+
+```bash
+git pull --rebase upstream master  # assuming upstream is the official repo
```

Comment:
This should be:
`git pull --rebase`
without `upstread master` part. otherwise it fails per https://github.com/pytorch/pytorch/issues/18388#issuecomment-476738007
Thanks.